### PR TITLE
 Add Math operations

### PR DESCRIPTION
**Pull request by Allatest**

### CODE REVIEW

File: js-operators-coercion/operators.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+// Завдання 2
+//let person1 = "Anna";
+//let person2 = "Alex";
+//let greetingForBoth = " Привіт, " + person1 + "і" + person2;
+// console.log(greetingForBoth);
+// let concatenatedString = person1 + person2 + greetingForBoth;
+// console.log(concatenatedString);
+
+
```
Comment (landriiuk) [inline js-operators-coercion/operators.js]: Why is this code commented on?

File: js-operators-coercion/string.js
Comment (landriiuk) [inline js-operators-coercion/string.js:1]: missed verification " - перевірте чи містить дана строка слово "world""

File: js-operators-coercion/number.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+//Завдання 3
+let myValue = "5.489";
+console.log(myValue);
+let myNumber = parseFloat(myValue);
+console.log(myNumber);
+let isNotNaN = !isNaN(myNumber);
+console.log(isNotNaN);
+myNumber = Math.round(myNumber * 10) / 10;
```
Comment (landriiuk) [inline js-operators-coercion/number.js:8]: you can use toFixed(1);

**Review by landriiuk — CHANGES_REQUESTED** [review]

**Comment by landriiuk** [comment]
There are a few moments that need to be finished or reworked. Pls finish and fix, then merge this pr

**Review by Allatest — COMMENTED** [review]
New changes added

**Pull request merged**